sophos_central_cloud/manifest.json

```diff
@@ -0,0 +1,40 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "7293cd88-ceda-4094-94cd-09851f203f0e",
+  "app_id": "sophos-central-cloud",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Gain insights into Sophos Central Cloud logs.",
+    "title": "sophos-central-cloud",
+    "media": [],
+    "classifier_tags": [
+      "Category::Log Collection",
+      "Category::Security",
+      "Submitted Data Type::Logs"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "auto_install": false,
+      "source_type_id": 18598661,
+      "source_type_name": "SophosCentralCloud",
```

Comment:
This should be "Sophos Central Cloud" with spaces

Reply:
Noted.